tests/pages/jobs/JobDetails-cy.js

```diff
@@ -84,9 +70,7 @@ xdescribe('Job Details', function () {
 
       cy.get('.page table .expanding-table-child')
         .should(function ($children) {
-          // Four table columns, two table rows, each with two children.
-          // 4 * 2 * 2 = 16
-          expect($children.length).to.equal(16);
+          expect($children.length).to.equal(20);
```

Comment:
The hidden rows only add 2 new rows, so if anything it would be 18 and not 20.
Good catch @mlunoe